icepyx/core/query.py

```diff
@@ -37,6 +37,8 @@ class GenQuery:
     spatial_extent : list of coordinates or string (i.e. file name)
         Spatial extent of interest, provided as a bounding box, list of polygon coordinates, or
         geospatial polygon file.
+        You can submit at most one bounding box or list of polygon coordinates.
+        Per NSIDC requirements, geospatial polygon files may only contain one feature (polygon).
         Bounding box coordinates should be provided in decimal degrees as
         [lower-left-longitude, lower-left-latitute, upper-right-longitude, upper-right-latitude].
```

Comment:
Maybe add some extra notes for the cross-dateline case. Specifically, state that `xdateline` should be set to `True` if lower-left longitude is positive (>0) but upper-right longitude is negative (<0)? Also think we should explicitly mention something about the (-180 to 180) or (0-360) range here. Here's a draft sentence:

```suggestion
        Bounding box coordinates should be provided in decimal degrees as
        [lower-left-longitude, lower-left-latitute, upper-right-longitude, upper-right-latitude].
        Note: longitude values are assumed to go from -180 to +180, with 0 being the Prime Meridian
        at Greenwich. If your region of interest crosses the International Date Line (i.e. lower-left
        longitude is positive (>0) but upper-right longitude is negative (<0), please set `xdateline=True`.
```